pages/12.Plugins/19.Zapier/docs.en.md

```diff
@@ -115,4 +115,6 @@ Even this integration is developed as an open source software as is usual with M
 
 You may encounter a 404 error during configuration. If this happens, [clear the cache][clear-cache].
 
-[clear-cache]: </troubleshooting#1-clear-the-cache>
+[cache]: </troubleshooting>
```

Comment:
Sorry if I was unclear on this one!

The macro used inline is [clear-cache] and the link was correct, so we need:

```suggestion
[clear-cache]: </troubleshooting#1-clear-the-cache>
```